assets/src/stories-editor/components/media-inserter/index.js

```diff
@@ -12,10 +12,11 @@ import { withDispatch, withSelect } from '@wordpress/data';
 import { DropdownMenu } from '@wordpress/components';
 import { compose, ifCondition } from '@wordpress/compose';
 import { __, sprintf } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
-import { processMedia } from '../../helpers';
+import { processMedia, isBlockAllowedOnPage } from '../../helpers';
```

Comment:
Why isn't the hook used here?

Reply:
This component hasn't been migrated to use hooks yet.

Reply:
I'll take a stab at rewriting it 👍

Reply:
Addressed in 144c430993400716171afc71b47df7bdfddfbc30

Reply:
:+1: